### ISSUE TITLE
Incorrect next version

### ISSUE THREAD

**Issue by domoritz** [python-semantic-release/python-semantic-release#484]
## The problem

I have a project with a latest release of `2.0.0-beta.1` and expect a new release of `2.0.0-beta.2`. However, I am getting `0.1.0-beta.1`. 

```
debug: get_new_version(2.0.0-beta.1, '0.0.0', 'minor', True, True)
debug: get_new_version -> 0.1.0-beta.1
```

## Expected behavior

```
debug: get_new_version(2.0.0-beta.1, '0.0.0', 'minor', True, True)
debug: get_new_version -> 2.0.0-beta.2
```

## Additional context

This is happening in https://github.com/cmudig/draco2. I made a manual release and added a tag to make sure semantic release has the right base version.

**Comment by codejedi365** [comment]
@domoritz, I'm sorry your issue never got attention two years ago but as a new maintainer I'm cleaning out the backlog.

I recommend trying out our latest version `v9.3.0` and seeing if your versioning issue still exists. PSR has come a long way since and I hope your issue would already be resolved.

**Comment by domoritz** [comment]
I get it. I've since released a 2.0 so I don't need this anymore. Feel free to close.

**Comment by codejedi365** [comment]
Thanks for the reply, let us know if you run into issues again in the future.
